wayshot/config.toml

```diff
@@ -0,0 +1,26 @@
+# base screenshot properties
+[screenshot]
+# output to take screenshot
+# output = "default"
+# should contain cursor?
+cursor = false
+# should copy screenshot to clipborad?
+clipboard = true
+# should write screenshot as file in filesystem?
+fs = true
```

Comment:
I think `fs` is not very user friendly. How about "save-to-disk"?

Reply:
I see, you are indeed correct. How about a `file` name then? The thing is, I want to keep config values naming as close as possible to cli arguments, so users will be able to know how to tweak config right away, if they were working with `wayshot`'s cli args before.

Reply:
Renamed to `file`

Reply:
Apologies, somehow missed this in actual code. Will push a fix later today.